HISTORY.md

```diff
@@ -14,6 +14,7 @@
 * EXPERIMENTAL: Add new API `DB::ClipColumnFamily` to clip the key in CF to a certain range. It will physically deletes all keys outside the range including tombstones.
 * Add `MakeSharedCache()` construction functions to various cache Options objects, and deprecated the `NewWhateverCache()` functions with long parameter lists.
 * Changed the meaning of various Bloom filter stats (prefix vs. whole key), with iterator-related filtering only being tracked in the new \*`_LEVEL_SEEK_`\*. stats. (#11460)
+* Add `WaitForCompact()` to wait for all flush and compactions jobs to finish. Jobs to wait include the unscheduled (queued, but not scheduled yet). `WaitForCompactOptions` has an option, `abort_on_pause`, to abort on pause. When set to true and `PauseBackgroundWork()` is called while or before waiting, `Status::Aborted` will be returned immediately.
```

Comment:
I still think there's too much detail on how to use it. The first sentence should suffice.